backend/controllers/Auth/RegisterController.js

```diff
@@ -0,0 +1,24 @@
+class RegisterController {
+    constructor(userService, httpStatusCodes) {
+        this.userService = userService;
+        this.httpStatusCodes = httpStatusCodes;
+    }
+
+    async invoke(req, res) {
+        const { email, password, password_confirmation } = req.body;
+
+        if ((!email, !password || !password_confirmation || password !== password_confirmation)) {
```

Comment:
are you sure there should be `comma`? plase add express-validator in some of next PR

Reply:
yes, I'll. I just add di, split controllers and add smaller improvements in this PR. Code is copied for now